Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.py

```diff
@@ -1032,9 +1032,9 @@ def update_remote_system_command(client, args):
 
             close_xdr_incident = argToBoolean(client._params.get("close_xdr_incident", True))
 
-            if not close_xdr_incident:
-                demisto.debug(f"Reverting to previous status {remote_args.data.get('status')}")
-                update_args['status'] = remote_args.data.get('status')
+            if not close_xdr_incident and (update_args.get('status') in XDR_RESOLVED_STATUS_TO_XSOAR):
```

Comment:
```suggestion
            if not close_xdr_incident and (update_args.get('status') in XDR_RESOLVED_STATUS_TO_XSOAR):
```
Shouldn't this be: XSOAR_RESOLVED_STATUS_TO_XDR?

Reply:
Fixed